oml/lightning/pipelines/logging.py

```diff
@@ -31,6 +35,66 @@ def prepare_tags(cfg: TCfg) -> List[str]:
     return tags
 
 
+class ClearMLLogger(Logger):
+    def __init__(self, **kwargs: Any):
+        try:
+            from clearml import Task
+        except ImportError as e:
+            raise ModuleNotFoundError(
+                "This contrib module requires clearml to be installed. "
+                "You may install clearml using: \n pip install clearml>=1.5.0 \n"
```

Comment:
let's avoid hardcoding the exact version here, I'm sure we will forget to adjust it after we change it in requirements :)

Reply:
Done ;)